superset/tags/commands/create.py

```diff
@@ -96,13 +94,10 @@ def run(self) -> None:
     def validate(self) -> None:
         exceptions = []
         # Validate object_id
-        if self._objects_to_tag:
-            if any(obj_id == 0 for obj_type, obj_id in self._objects_to_tag):
-                exceptions.append(TagInvalidError())
-
+        if objects_to_tag := self._properties.get("objects_to_tag", []):
             # Validate object type
             skipped_tagged_objects: list[tuple[str, int]] = []
-            for obj_type, obj_id in self._objects_to_tag:
+            for obj_type, obj_id in objects_to_tag:
                 skipped_tagged_objects = []
```

Comment:
why is skipped_tagged_objects defined twice, once above, and then again in this for loop?